pdq/DMS.go

```diff
@@ -0,0 +1,18 @@
+package pdq
+
+import (
+	"fmt"
+	"math"
+)
+
+func CalcDMS(abstractness float64, instability float64) (float64, error) {
+	if abstractness < 0 || instability < 0 {
+		return -1, fmt.Errorf("abstractness %v, instability %v; values must be >= 0", abstractness, instability)
+	}
+
+	if abstractness >= 1e308 || instability >= 1e308 {
```

Comment:
func checks the input values ​​for negativity and extremely large values but instead of checking for a specific large number like **1e308**, betterto use a predefined `math.Inf` const that represents infinity